trigger/cmds.py

```diff
@@ -380,10 +424,19 @@ def parse(self, results, device):
             list
 
         :param device:
-            A `~trigger.netdevices.NetDevice` object
+            Device object
+        :type device:
+            `~trigger.netdevices.NetDevice`
+
+        :param commands:
+            (Optional) A list of commands to execute on the device. If not
+            specified in they will be inherited from commands passed to the
+            class constructor.
+        type commands:
+            list
         """
         func = self._lookup_method(device, method='parse')
-        return func(results, device)
+        return func(results, device, commands)
```

Comment:
For watchers, this is the part of this commit you care about. Sorry about the pep8 stuff!